jte/src/main/java/gg/jte/compiler/TemplateCompiler.java

```diff
@@ -298,7 +264,8 @@ public ClassInfo generateTemplateCall(String name, LinkedHashSet<ClassDefinition
         CodeGenerator codeGenerator = createCodeGenerator(classInfo, classDefinitions, templateDependencies);
         new TemplateParser(code, TemplateType.Template, codeGenerator, config, codeResolver).parse();
 
-        classDefinition.setCode(codeGenerator.getCode(), codeGenerator.getBinaryTextParts());
+        classDefinition.setCode(codeGenerator.getCode(), codeGenerator.getBinaryTextParts(), codeGenerator.getParamInfo(), codeGenerator.getImports());
+        classDefinition.setParams(codeGenerator.getParamInfo());
```

Comment:
I think the setParams() call is redundant here. Maybe we can remove that method?

Reply:
Yes, missed this in changing the above method.

Reply:
Done